src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -53,14 +61,41 @@ export default class DefaultContentShareController implements ContentShareContro
 
   pauseContentShare(): void {
     this.mediaStreamBroker.toggleMediaStream(false);
+    this.forEachContentShareObserver(observer => {
```

Comment:
Ideally, these observer events would only fire if the state actually transitioned. For example, if you call pauseContentShare twice, this will generate two pause events, but the second one is redundant. If I call pauseContentShare, then I know that because I called it that I'm going to get a callback saying I paused... it's not telling me anything I didn't already know.